src/picongpu/include/particles/flylite/types/Superconfig.hpp

```diff
@@ -0,0 +1,50 @@
+/** Copyright 2017 Axel Huebl
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "pmacc_types.hpp"
+#include "math/Vector.hpp"
+
+
+namespace picongpu
+{
+namespace flylite
+{
+namespace types
+{
+    /** Ion Superconfiguration
+     *
+     * This is the attribute type for an ion's hydrogenic superconfiguration.
```

Comment:
yes but not in collisional-radiative modeling, there it refers to screened hydrogen models. will clarify